ops/terraform/modules/resources/bfd_pipeline/main.tf

```diff
@@ -1,4 +1,17 @@
 locals {
+  # TODO: temporary work around to support dynamic rds writer node resolution.
+  #       This would be more reasonably encoded as inputs to the parent module, but that's less
+  #       clear with the disconnect between stateful and stateless modules. Near-term adoption of
+  #       a terraservices strategy will make this more obvious.
+  environments = {
+    test = {
+      rds_cluster_identifier = "bfd-1652-v70-pre-synthea-load" # TODO: Temporary. To be removed after BFD-1746
```

Comment:
Should be noted in that ticket to remove this, so we don't forget.

Reply:
Excellent. I've added it to the AC.